code/modules/mob/living/carbon/human/human.dm

```diff
@@ -67,7 +67,6 @@
 	h_style = "Bald"
 	..(new_loc, "Golem")
 	gender = NEUTER
-	meat_type = /obj/item/weapon/ore/diamond
```

Comment:
I removed this since the meat_type of a `/human` is now governed by their species datum.
I've defined golems' `meat_type` in `/datum/species/golem`.